spec/abilities/atmosphere/appliance_ability_builder_spec.rb

```diff
@@ -51,7 +51,7 @@
   def start_appl(type, at, pdp_method_sym = nil, pdp_response = nil)
     appl = build(:appliance, appliance_set: build_as(type), appliance_type: at)
     if pdp_method_sym
-      allow(pdp).to receive(pdp_method_sym).with(at).and_return(pdp_response)
+      allow_any_instance_of(Atmosphere::DefaultPdp).to receive(pdp_method_sym).with(at).and_return(pdp_response)
```

Comment:
Please try to improve line 10 to return pdp double instead of allowing any action on DeafultPdp. Something like:

```ruby
let(:pdp) { double('pdp') } 
let(:pdp_class) do 
  pdp_class = double('pdp_class')
  allow(pdp_class).to receive(:new).and_return(pdp)
  pdp_class
end
let(:ability) { Atmosphere::Ability.new(user, true, pdp_class) }
```